test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -90,26 +83,28 @@ def test_single_mom_hook_failure_affects_vnode(self):
         Once offlined, reset the mom by issueing pbsnodes -r
         and check if the job runs on one of the vnodes.
         """
-        self.create_multi_vnodes(1)
+        single_mom = ""
+        if len(self.moms.values()) != 1:
+            self.server.manager(MGR_CMD_DELETE, NODE, id="@default",
+                                expect=True)
+        single_mom = self.moms.values()[0]
+

```

Comment:
Do we need to add the following code in case the mom provided has an unwanted vnodedef file (sorry I can't get the formatting of the comment right)
```
if self.moms.values()[0].has_vnode_defs():
       self.moms.values()[0].delete_vnode_defs()
```

Reply:
I think so, this is just to safe guard that the two moms being added do not come with already existing vnode definitions.

Reply:
I meant do we need to add that code to the single mom test case.  :)

Reply:
oh ok, I get it. I can add it, but if you are okay I'd like not to touch the single node test. The test is very delicate and is now working on all 3 platforms (cray, uv and normal linux). I'm worried that changing it may cause trouble on one or more of these platforms.

Reply:
Since it's working then it seems the change is not necessary.  I certainly don't want to upset the delicate balance if it is working.  :)  Thanks for the response.